lessons/advanced/behaviours.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Behaviours
+category: advanced
+order: 
+lang: en
+---
+
+We learned about Typespecs the previous lesson, here we'll learn about how to require a module implement those specifications.  In Elixir, this functionality is referred to as Behaviours.
+
+{% include toc.html %}
+
+## Uses
+
+Behaviours give us the ability to define a set of functions a module must implement and ensure that set has been implemented, this makes it easy to create and share a public API.
```

Comment:
I guess this part can be simplified - I had to read it couple times to understand. :)